{"name": "test_ChangingBalance", "status": "failed", "statusDetails": {"message": "AssertionError: Изменений между DataFrame не найдено\nassert not True\n +  where True = Empty DataFrame\\nColumns: []\\nIndex: [].empty", "trace": "def test_ChangingBalance():\n        df1 = pd.read_csv('UsersWithPositiveBalance.csv', nrows=2)\n        df2 = pd.read_csv('ChangedBalance.csv')\n    \n        assert df1.shape[0] == df2.shape[0], \"Количество строк в DataFrame не совпадает\"\n    \n        assert set(df1.columns) == set(df2.columns), \"Столбцы в DataFrame не совпадают\"\n    \n        df1 = df1.sort_values('msisdn').reset_index(drop=True)\n        df2 = df2.sort_values('msisdn').reset_index(drop=True)\n    \n        df2 = df2[df1.columns]\n        df2 = df2.reindex(df1.index)\n        df = df1.compare(df2)\n    \n>       assert not df.empty, \"Изменений между DataFrame не найдено\"\nE       AssertionError: Изменений между DataFrame не найдено\nE       assert not True\nE        +  where True = Empty DataFrame\\nColumns: []\\nIndex: [].empty\n\nTest_e2eExactlyMinute.py:133: AssertionError"}, "start": 1746269935640, "stop": 1746269935653, "uuid": "676b87d4-51bb-4705-b8e0-bc1af7eefbd6", "historyId": "dba562804614f4713c52f2461a5ace1e", "testCaseId": "dba562804614f4713c52f2461a5ace1e", "fullName": "Test_e2eExactlyMinute#test_ChangingBalance", "labels": [{"name": "suite", "value": "Test_e2eExactlyMinute"}, {"name": "host", "value": "DESKTOP-S3HFG4F"}, {"name": "thread", "value": "9772-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Test_e2eExactlyMinute"}]}